Android.mk

```diff
@@ -34,6 +34,10 @@ ifeq ($(TARGET_USE_TRUSTY),true)
     KERNELFLINGER_CFLAGS += -DUSE_TRUSTY
 endif
 
+ifeq ($(TARGET_USE_OPTEE),true)
```

Comment:
ivshmem is not bound to OPTEE. suggest to change the TARGET_USE_OPTEE to TARGET_USE_IVSHMEM